andresSaldanaAguilar/react-certification-2021
# Core concepts and styling

#### What does this PR do?

Fulfills the tasks for the Mini-Challenge 1: Core Concepts and Styling

#### How should this be manually tested?

You can open the Codesandbox pointing to this branch and you will able to see the app running

#### Checklist

- [x] I added the menu component
- [x] I redid the home view and inject the mocks into it
- [x] I added a not found landing page
- [x] I styled the components with the styled-components library
